js/views/modals/wallet/SendMoney.js

```diff
@@ -189,6 +201,12 @@ export default class extends baseVw {
       this._$sendConfirm = null;
 
       this.$('#walletSendCurrency').select2();
+
+      const fee = this.sendConfirmBox && this.sendConfirmBox.getState().fee;
```

Comment:
If sometimes this.sendConfirmBox exists, shouldn't the view be removed before creating a new one below?